src/types/Links.ts

```diff
@@ -0,0 +1,5 @@
+export enum LinkType {
```

Comment:
```suggestion
export enum Routes {
```

Reply:
I had a conflict with Reacts Router, so I renamed the enum as  PageRoutes.